feat: add Android Authenticator docs (#3705)
aws-amplify/amplify-ui: docs/src/pages/[platform]/connected-components/authenticator/customization/customization.labels-and-text.android.mdx

> Using the same techniques as [Internationalization (I18n)](#internationalization-i18n), you can customize the labels (hints) and text of the components.  Every string defined by the library is prefixed with `authenticator_` and strings adhere to a hierarchical naming convention to help identify its usage.  All such strings can be found in the `authenticator/src/main/res/values` folder.

Reviewer: ```suggestion
Using the same techniques as [Internationalization (I18n)](#internationalization-i18n), you can customize the labels (hints) and text of the components.  Every string defined by the library is prefixed with `amplify_ui_authenticator_` and strings adhere to a hierarchical naming convention to help identify its usage.  All such strings can be found in the `authenticator/src/main/res/values` folder.
```

> Using the same techniques as [Internationalization (I18n)](#internationalization-i18n), you can customize the labels (hints) and text of the components.  Every string defined by the library is prefixed with `amplify_ui_authenticator_` and strings adhere to a hierarchical naming convention to help identify its usage.  All such strings can be found in the `authenticator/src/main/res/values` folder.

Reviewer 2: ```suggestion
Using the same techniques as [Internationalization (I18n)](#internationalization-i18n), you can customize the labels (hints) and text of the components. Every string defined by the library is prefixed with `amplify_ui_authenticator_` and strings adhere to a hierarchical naming convention to help identify its usage.  All such strings can be found in the `authenticator/src/main/res/values` folder.
```

For example, here's a string from the library:
```xml

> <string name="authenticator_field_warn_already_exists">%s already exists</string>

Reviewer: ```suggestion
<string name="amplify_ui_authenticator_field_warn_existing">%s already exists</string>
```

```

You can change this string by overriding the string in your application's default resources:
```xml

> <string name="authenticator_field_warn_already_exists">Sorry, this is already taken</string>

Reviewer: ```suggestion
<string name="amplify_ui_authenticator_field_warn_existing">Sorry, this is already taken</string>
```
